packages/ios-client/Sources/FishjamClient/CommandsQueue.swift

```diff
@@ -0,0 +1,49 @@
+import Promises
+
+internal class CommandsQueue {
+    var clientState: ClientState = ClientState.CREATED
+    private var commandsQueue: [Command] = []
+
+    @discardableResult
+    func addCommand(_ command: Command) -> Promise<Void> {
+        commandsQueue.append(command)
+        if commandsQueue.count == 1 {
+            command.execute()
+        }
+        return command.promise
+    }
+
+    func finishCommand() {
+        guard !(commandsQueue.isEmpty) else { return }
+        let command = commandsQueue.first!
```

Comment:
**suggestion:** You could just write:
`guard let command = commandsQueue.first else { return }`